### PR TITLE
Add gradle commands

### PR DESCRIPTION
**Pull request by lagelalegal**

### CODE REVIEW

File: pages/common/gradle.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Gradle
+
+> Gradle is the official build system for Android Studio
+
+- Show version
+
+`gradle -v`
+
+- Complie a package(Make sure you have a gradle file in the workspace)
```
Comment (ostera) [inline pages/common/gradle.md]: I think the comment in parens is unnecessary. Also you have a typo in `Complie`.

File: pages/common/gradle.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Gradle
+
+> Gradle is the official build system for Android Studio
+
+- Show version
+
+`gradle -v`
```
Comment (ostera) [inline pages/common/gradle.md]: I think this too is unnecessary

**Comment by lagelalegal** [comment]
@ostera , I got it, thank you very much.

**Comment by ostera** [comment]
Looks alright to me :+1:

**Comment by igorshubovych** [comment]
LGTM. Thank you!

**Pull request merged**